components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodClientProjectSessionTracker.kt

```diff
@@ -120,7 +120,9 @@ class GitpodClientProjectSessionTracker(
 
         // Ignore ports that aren't actually used by the user (e.g. ports used internally by JetBrains IDEs)
         val backendPort = BuiltInServerManager.getInstance().waitForStart().port
-        val serverPort = StartupUtil.getServerFuture().await().port
```

Comment:
ℹ️ This change is because `com.intellij.idea.StartupUtil` doesn't exist anymore in 2022.3.